types/state.go

```diff
@@ -60,8 +66,17 @@ func NewFromGenesisDoc(genDoc *types.GenesisDoc) (State, error) {
 		return State{}, fmt.Errorf("error in genesis doc: %w", err)
 	}
 
-	if len(genDoc.Validators) != 1 {
```

Comment:
why remove this enforcement? do you expect to have `nil` at any point?

Reply:
because the actual valset will be checked again after apply abci init chain.
In this line :  
https://github.com/decentrio/rollkit/blob/5d6559b2f7ee0c7fba15a4874ed21252b2645e4b/block/manager.go#L1165